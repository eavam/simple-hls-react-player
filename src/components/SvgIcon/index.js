import React from 'react'
import PropTypes from 'prop-types'

const SvgIcon = ({ viewBox, className, children }) => 
  <svg viewBox={viewBox} className={className}>
    {children}
  </svg>

SvgIcon.propTypes = {
  viewBox: PropTypes.string,
  className: PropTypes.string,
}

export default SvgIcon