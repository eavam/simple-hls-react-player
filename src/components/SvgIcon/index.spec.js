import React from 'react'
import renderer from 'react-test-renderer'
import SvgIcon from './index.js'

test('SvgIcon', () => {

  const props = {
    viewBox: '',
    className: '',
    children: <div />
  }

  const componentDisabled = renderer.create(
    <SvgIcon {...props} />
  )
  let tree = componentDisabled.toJSON()
  expect(tree).toMatchSnapshot()

})