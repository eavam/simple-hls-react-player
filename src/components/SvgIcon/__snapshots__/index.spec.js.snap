// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SvgIcon 1`] = `
<svg
  className=""
  viewBox=""
>
  <div />
</svg>
`;
